cmd/crossplane/main.go

```diff
@@ -36,7 +36,7 @@ func main() {
 
 	// Re-sync resources every minutes
 	// TODO: 1 minute could be too aggressive - we will update this in the future.
-	syncPeriod := 1 * time.Minute
+	syncPeriod := 60 * time.Minute
```

Comment:
Was this supposed to be included in this change?

Reply:
Ah.. no, good catch.